themes/default/content/blog/esc-kubernetes-cluster-and-app/index.md

```diff
@@ -0,0 +1,317 @@
+ +      ├─ aws:ec2:SecurityGroup            eks-cluster-nodeSecurityGroup                  created (1s)
+ +      ├─ pulumi:providers:kubernetes      eks-cluster-eks-k8s                            created (0.14s)
+ +      ├─ eks:index:VpcCni                 eks-cluster-vpc-cni                            created (2s)
+ +      ├─ kubernetes:core/v1:ConfigMap     eks-cluster-nodeAccess                         created (0.60s)
+ +      ├─ aws:ec2:SecurityGroupRule        eks-cluster-eksExtApiServerClusterIngressRule  created (1s)
+ +      ├─ aws:ec2:SecurityGroupRule        eks-cluster-eksClusterIngressRule              created (1s)
+ +      ├─ aws:ec2:SecurityGroupRule        eks-cluster-eksNodeInternetEgressRule          created (1s)
+ +      ├─ aws:ec2:SecurityGroupRule        eks-cluster-eksNodeClusterIngressRule          created (2s)
+ +      ├─ aws:ec2:SecurityGroupRule        eks-cluster-eksNodeIngressRule                 created (2s)
+ +      ├─ aws:ec2:LaunchConfiguration      eks-cluster-nodeLaunchConfiguration            created (1s)
+ +      ├─ aws:cloudformation:Stack         eks-cluster-nodes                              created (108s)
+ +      └─ pulumi:providers:kubernetes      eks-cluster-provider                           created (0.12s)
+
+Outputs:
+    kubeconfig: {
+        apiVersion     : "v1"
+        clusters       : [
+            [0]: {
+                cluster: {
+                    certificate-authority-data: "<redacted>"
+                    server                    : "https://FB9978041F7B72B227B7E95F076822E4.gr7.us-west-2.eks.amazonaws.com"
+                }
+                name   : "kubernetes"
+            }
+        ]
+        contexts       : [
+            [0]: {
+                context: {
+                    cluster: "kubernetes"
+                    user   : "aws"
+                }
+                name   : "aws"
+            }
+        ]
+        current-context: "aws"
+        kind           : "Config"
+        users          : [
+            [0]: {
+                name: "aws"
+                user: {
+                    exec: {
+                        apiVersion: "client.authentication.k8s.io/v1beta1"
+                        args      : [
+                            [0]: "eks"
+                            [1]: "get-token"
+                            [2]: "--cluster-name"
+                            [3]: "eks-cluster-eksCluster-1268ff9"
+                        ]
+                        command   : "aws"
+                        env       : [
+                            [0]: {
+                                name : "KUBERNETES_EXEC_INFO"
+                                value: (json) {
+                                    apiVersion: "client.authentication.k8s.io/v1beta1"
+                                }
+
+                            }
+                        ]
+                    }
+                }
+            }
+        ]
+    }
+
+Resources:
+    + 27 created
+
+Duration: 9m38s
+```
```

Comment:
Maybe show a quick `kubectl get pods` or something to show the cluster is running? I think you show this later, but as a reader, it's probably a more natural "next step" than deploying an entirely new application.